src/pe/debug.rs

```diff
@@ -150,8 +150,8 @@ impl<'a> CodeviewPDB70DebugInfo<'a> {
 
         // calculate how long the eventual filename will be, which doubles as a check of the record size
         let filename_length = idd.size_of_data as isize - 24;
-        if filename_length < 0 || filename_length > 1024 {
-            // the record is too short or too long to be plausible
+        if filename_length < 0 {
```

Comment:
Is there no maximum bound to check against ? (Worried about regressions if removing this)

also curious the number 1024 was used, I wonder why that was chosen ? (Haven’t looked at this code in a while)

Reply:
1024 is an odd choice - MAX_PATH + some small constant factor (to include the hash and other data in the section) seems reasonable, but 1024 is a bit bigger than I'd expect for the typical size of the directory.

That said, I don't think there is a maximum bound - and as this crate is typically used, the memory is already mapped/allocated, so it's mostly harmless to remove the check.